src/Microsoft.AspNetCore.Mvc/project.json

```diff
@@ -16,6 +16,7 @@
     "Microsoft.AspNetCore.Mvc.Formatters.Json": "1.0.0-*",
     "Microsoft.AspNetCore.Mvc.Localization": "1.0.0-*",
     "Microsoft.AspNetCore.Mvc.Razor": "1.0.0-*",
+    "Microsoft.AspNetCore.Mvc.TagHelpers": "1.0.0-*",
```

Comment:
> Before now though, Tag Helpers didn't need their own services in DI

They did, `IMemoryCache`, which was registered by MVC _by chance_. Now it's `IMemoryCache` and `IHtmlFragmentCache` which are also registered by MVC, on purpose for this feature. So it doesn't change, nothing more to register to use `CacheTagHelper` from MVC.

Before you had to:
- Reference TagHelper package
- Import the tag helpers in the view

And now it's
- Import the tag helpers in the view

And _Optionally_ register `IDistributedCache` if you want to use it